react/src/pages/dashboard/ParticipantTable.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { useState } from 'react';
+import { makeStyles, Chip, IconButton, Typography } from '@material-ui/core';
+import { Cancel, FileCopy } from '@material-ui/icons';
+import MaterialTable, { MTableToolbar } from 'material-table';
+import { Participant, rows } from './MockData';
+import DatasetTypes from './DatasetTypes';
+
+const useStyles = makeStyles(theme => ({
+    chip: {
+        color: "primary",
+        marginRight: '10px',
+        colorPrimary: theme.palette.primary,
+    },
+    copyIcon: {
+        marginLeft: theme.spacing(1),
+    }
+}));
+
+const organizeDatasetTypes = (types: string[]) => {
+    const newTypes: {[key: string]: number} = {};
+    types.map(type => {
+        if(newTypes.hasOwnProperty(type)){
+            newTypes[type] += 1;
+        }else{
+            newTypes[type] = 1;
+        }
+    })
+    return newTypes
+}
```

Comment:
There was a type warning because the return value of map is meant to be used (as opposed to forEach) so this should be a forEach loop or using Array.reduce:
```suggestion
const organizeDatasetTypes = (types: string[]) => types.reduce<Hash>(
    (newTypes, type) => {
        if (newTypes[type]) {
            newTypes[type] += 1;
        } else {
            newTypes[type] = 1;
        }
        return newTypes;
    }, Object.create(null));
```